GooglePlacesAutocomplete.js

```diff
@@ -642,8 +642,24 @@ export const GooglePlacesAutocomplete = forwardRef((props, ref) => {
     );
   };
 
-  const _onBlur = () => {
+  const isNewFocusInAutocompleteResultList = ({ relatedTarget, currentTarget }) => {
+    if (!relatedTarget) return false;
+
+    var node = relatedTarget.parentNode;
+
+    while (node) {
+      if (node.id === 'react-native-google-places-autocomplete-result-list-id') return true;
```

Comment:
can we rename this to something a little shorter

Reply:
done